lib/checks/base_paths_missing_from_publishing_api.rb

```diff
@@ -0,0 +1,31 @@
+module Checks
+  class BasePathsMissingFromPublishingApi
+
+    def initialize(checker_db, whitelist)
+      @checker_db = checker_db
+      @whitelist = whitelist
+    end
+
+    # find base_paths present in Rummager which don't map to a published content item in the Publishing API
+
+    def run_check
+      query = <<-SQL
+      SELECT
+      rc.base_path,
```

Comment:
A general style observation - I find it's a lot easier to read and understand these queries when these abbreviations are fully expanded into their respective table names. Saves having to remember what they are / jump back to the line containing the alias.